src/GitHub.App/Services/PullRequestService.cs

```diff
@@ -309,7 +309,7 @@ public IObservable<int> GetPullRequestForCurrentBranch(ILocalRepositoryModel rep
                 string mergeBase = await gitClient.GetPullRequestMergeBase(repo, remote.Name, baseSha, headSha, baseRef, pullRequest.Number);
                 if (mergeBase == null)
                 {
-                    throw new FileNotFoundException($"Couldn't find merge base between {baseSha} and {headSha}.");
+                    throw new InvalidOperationException($"Couldn't find merge base between {baseSha} and {headSha}.");
```

Comment:
In #1036, I changed this to `LibGit2Sharp.NotFoundException` so might be worth reverting this change until that's merged to prevent merge conflicts.